tests/backup/backup_upgrade_test.go

```diff
@@ -204,3 +205,366 @@ var _ = Describe("{StorkUpgradeWithBackup}", func() {
+			log.Infof("Schedule policy [%s] uid: [%s]", schedulePolicyName, schedulePolicyUid)
+		})
+		Step("Create schedule backup for each namespace", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			createSingleNSBackupTask := func(appName string) {
+				namespace := appNamespaces[appName][0]
+				labelSelectors := make(map[string]string)
+				singleNSScheduleName := fmt.Sprintf("%s-%s-single-ns-schedule-%v", BackupNamePrefix, namespace, time.Now().Unix())
+				log.InfoD("Creating schedule backup with schedule [%s] of namespace [%s]", singleNSScheduleName, namespace)
+				err = CreateScheduleBackup(singleNSScheduleName, SourceClusterName, backupLocationName, backupLocationUid, []string{namespace},
+					labelSelectors, orgID, preRuleNames[appName], preRuleUids[appName], postRuleNames[appName], postRuleUids[appName], schedulePolicyName, schedulePolicyUid, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule [%s]", singleNSScheduleName))
+				firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, singleNSScheduleName, orgID)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching name of the first schedule backup with schedule [%s]", singleNSScheduleName))
+				log.Infof("First schedule backup name: [%s]", firstScheduleBackupName)
+				singleNamespaces = SafeAppend(&mutex, singleNamespaces, namespace).([]string)
+				singleNSScheduleNames = SafeAppend(&mutex, singleNSScheduleNames, singleNSScheduleName).([]string)
+			}
+			_ = TaskHandler(Inst().AppList, createSingleNSBackupTask, Parallel)
+		})
+		Step("Create schedule backup of all namespaces", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			allNamespaces = make([]string, 0)
+			for _, appName := range Inst().AppList {
+				allNamespaces = append(allNamespaces, appNamespaces[appName]...)
+			}
+			allNSScheduleName = fmt.Sprintf("%s-all-ns-schedule-%v", BackupNamePrefix, time.Now().Unix())
+			labelSelectors := make(map[string]string)
+			log.InfoD("Creating schedule backup with schedule [%s] of all namespaces [%s]", allNSScheduleName, allNamespaces)
+			err = CreateScheduleBackup(allNSScheduleName, SourceClusterName, backupLocationName, backupLocationUid, allNamespaces,
+				labelSelectors, orgID, "", "", "", "", schedulePolicyName, schedulePolicyUid, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule [%s]", allNSScheduleName))
+			firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, allNSScheduleName, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching name of the first schedule backup with schedule [%s]", allNSScheduleName))
+			log.Infof("First schedule backup name: [%s]", firstScheduleBackupName)
+		})
+		Step("Upgrading px-backup", func() {
+			log.InfoD("Upgrading px-backup to latest version [%s]", latestPxBackupVersion)
```

Comment:
Let's make `latestPxBackupVersion` a env variable.

Reply:
Done. Env variable LATEST_PXBACKUP_VERSION can be used to set latestPxBackupVersion.